src/views/router/routes.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Chat from '../chat/chat'
+import { Home } from '../home'
+import Settings from '../settings/settings'
+import { FaCog, FaComment, FaHome } from 'react-icons/fa'
+
+/* eslint-disable sort-keys */
+export const appRoutes = {
+	Home: {
+		component: Home,
+		icon: <FaHome />,
+		label: 'Home',
+	},
+	Chat: {
+		component: Chat,
+		icon: <FaComment />,
+		label: 'Chat',
+	},
+	Settings: {
+		component: Settings,
+		icon: <FaCog />,
+		label: 'Settings',
+	},
+}
+/* eslint-enable sort-keys */
+export type Route = {
+	label: string
+	component: () => JSX.Element
+	icon: React.ReactNode
+}
```

Comment:
I'm pretty sure we can pass the Icons in the same exact was at the Components (they are both React components, just being passed in different ways)

```suggestion
export const appRoutes = {
	Home: {
		component: Home,
		icon: FaHome ,
		label: 'Home',
	},
	Chat: {
		component: Chat,
		icon: FaComment ,
		label: 'Chat',
	},
	Settings: {
		component: Settings,
		icon: FaCog ,
		label: 'Settings',
	},
}
/* eslint-enable sort-keys */
export type Route = {
	label: string
	component: () => JSX.Element
	icon: () => JSX.Element
}
```

Reply:
Resolving this. Will open another Review with my suggested changes.